src/api-reference/expense/expense-report/expense-form-field.markdown

```diff
@@ -3,9 +3,6 @@ title: Expense Form Field
 layout: reference
 ---
 
```

Comment:
The TOC should have: Get a list of form fields, Request, Response, Response Schema, Example Request, Example Response